node/lib/Logger.ts

```diff
@@ -72,15 +69,43 @@ export class Logger implements ILogger {
         this.logger.debug(msg, logObject);
     }
 
-    public getConfiguredTransports(): Array<any> {
-        return this.transportList;
+    public event(name: string, duration: number, data?: { [key: string]: any; }): void {
+        this.callAllTransports(t => t.event(name, duration, data));
+    }
+
+    public trace(message: string, traceLevel: TraceLevel, data?: { [key: string]: any; }): void {
+        this.callAllTransports(t => t.trace(message, traceLevel, data));
+    }
+
+    public metric(
+        name: string,
+        value: number,
+        count?: number,
+        min?: number,
+        max?: number,
+        data?: { [key: string]: any; }): void {
+        this.callAllTransports(t => t.metric(name, value, count, min, max, data));
+    }
+
+    public exception(exception: Error, data?: { [key: string]: any; }): void {
+        this.callAllTransports(t => t.exception(exception, data));
     }
 
-    public normalizeWith(normalizer: (logObject: any) => any): ILogger {
+    public getConfiguredTransports(): Array<string> {
+        return this.logger._names;
+    }
+
+    public normalizeWith(normalizer: (logObject: any) => any): Logger {
```

Comment:
Doesn't look like it was used anywhere, removed.

---
In reply to: [109506842](https://github.com/openT2T/opent2t/pull/40#discussion_r109506842) [](ancestors = 109506842)